src/factories.ts

```diff
@@ -410,7 +410,15 @@ export const createMention = (
  * @param content - The file content to be hashed
  * @returns An ActivityContentHash containing the keccak256 proof of the content
  */
-export const createHash = (content: string): ActivityContentHash => ({
+export const createHashWithContent = (content: string | Uint8Array): ActivityContentHash => ({
```

Comment:
Added a uint8array type here since we wanted the ability to hash byte arrays for things like audio files